packages/features/ee/workflows/lib/reminders/whatsappReminderManager.ts

```diff
@@ -269,4 +269,6 @@ export const scheduleWhatsappReminder = async (args: ScheduleTextReminderArgs) =
   }
 };
 
-export const deleteScheduledWhatsappReminder = deleteScheduledSMSReminder;
```

Comment:
`deleteScheduledSMSReminder` is being imported from `smsReminderManager` the way the import was happening was sometimes it wouldn't be fully imported if called.